packages/release-please/src/release-please.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { GitHubAPI } from 'probot/lib/github';
+import { ReleaseType } from 'release-please/build/src/release-pr';
+import { JavaYoshi } from 'release-please/build/src/releasers/java-yoshi';
+
+interface ConfigurationOptions {
+  strategy?: string;
+  primaryBranch: string;
+  releaseLabels: string[];
+}
+
+const DEFAULT_API_URL = 'https://api.github.com';
+const WELL_KNOWN_CONFIGURATION_FILE = '.github/release-please.json';
```

Comment:
I notice that here we're using `json`, but in the blunderbuss bot we're using `yaml`.  Can we pick one and be consistent?